### ISSUE TITLE
Verification of Face live(like face lock)

### ISSUE THREAD

**Issue by jai38** [justadudewhohacks/face-api.js#731]
can we just store details of one person details and then verify if the person is same or other? just like face lock work?

**Comment by josiahbryan** [comment]
Yes. Have you looked at the `FaceMatcher` class in this project...? Check the docs for that class.

 I wrote my own for finding "known" faces in video feeds for the purposes of switching cameras for streaming, but the idea is the same - get a face descriptor, store it, label it, then later match new descriptors from found faces against the existing descriptors you have stored. If under some threshold of Euclidean distance, then voila, you have a match.
